fixed URL for Use a custom log parser

ErikMoreau opened the pull request:
https://docs.microsoft.comcommit/cloud-app-security/custom-log-parser => https://docs.microsoft.com/cloud-app-security/custom-log-parser

azurecla commented:
Hi __@omartin2010__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (omartin). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

PRmerger commented:
@omartin2010 : Thanks for your contribution to the Azure documentation! The author, @cherylmc, has been notified to review your proposed change.

cherylmc commented:
Hi @omartin2010, already fixed. :-) I put them both through at pretty much the same time, based on your earlier recommendation. Thanks for making sure, though!

cherylmc commented:
#please-close

ErikMoreau committed:
fixed URL for Use a custom log parser

https://docs.microsoft.comcommit/cloud-app-security/custom-log-parser => https://docs.microsoft.com/cloud-app-security/custom-log-parser
